docs/Using-Fleet/FAQ.md

```diff
@@ -318,3 +336,27 @@ Run `sudo apt remove fleet-osquery -y`
 
 ### CentOS
 Run `sudo rpm -e fleet-osquery-X.Y.Z.x86_64`
+
+## How does Fleet determines online and offline status?
+
+### Online hosts
+
+**Online** hosts are devices that are connected to Fleet and will respond to a live query.
```

Comment:
```suggestion
**Online** hosts will respond to a live query.
```

I think we can simplify this. This is because some users may be confused by "Online hosts are devices that are connected to Fleet." I anticipate questions like "Aren't offline hosts still connected?"